### ISSUE TITLE
Highly conductive finite difference material severe should cause fatal

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#6081]
Error message:

```
   ** Severe  ** InitialInitHeatBalFiniteDiff: Found Material that is too thin and/or too highly conductive, material name = PLASTIC
   **   ~~~   ** High conductivity Material layers are not well supported by Conduction Finite Difference,  material conductivity = 2.000 [W/m-K]
```

It's a severe, and per the user, it should go ahead and fatal.

**Comment by mitchute** [comment]
@Myoldmopar I'm going to go ahead and make this change. Any objections @mjwitte , or anyone else?

**Comment by mjwitte** [comment]
@mitchute I suppose it's ok.  Will be interesting to see if any of the fd example files fail with it.  The fatal error needs to inform what the threshold is.

**Comment by Myoldmopar** [comment]
Closed via #6235

**Comment by sajithjay** [comment]
How to avoid this issue in BEopt when updating an old BEopt case (2.3.0.2) in a new version (2.8)? Has there been a discussion?
